docs/gen-2/gold-silver/main/glitchless/resources/README.md

```diff
@@ -0,0 +1,7 @@
+# Pokemon Gold-Silver Resources
+
+* [Don Manip](don-manip.md)
+* [LID Frames](lid-frames.md)

```

Comment:
This should be `LID-frames.md`.

Instead of changing this name however, I suggest changing the name of the file to `lid-frames.md`